src/style.css

```diff
@@ -1,3 +1,58 @@
+/* stylelint-disable declaration-block-no-redundant-longhand-properties */
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+}
+
+img {
+  width: 40px;
+  height: 40px;
+  top: 10px;
+  left: 50px;
```

Comment:
The 'top' and 'left' properties require the position to be other than 'static' to have any effect. If you want to position the image, ensure that you set its position property appropriately (e.g., 'position: relative;' or 'position: absolute;' with a parent having 'position: relative;').